client_plugin/drivers/shared/dockerops/dockerops.go

```diff
@@ -335,6 +337,45 @@ func (d *DockerOps) isFileServiceRunning(servID string, volName string) (uint32,
 	return port, true
 }
 
+// getServiceIDAndPort - return the file service ID and port for given volume
+// It takes some time from service being brought up to a
```

Comment:
This message is from the function isFileserverRunning(). Write a different message explaining this function.